internal/examples-shared-ui/src/lib/example-scenario.component.scss

```
@@ -365,7 +365,8 @@
   form {
     display: flex;
     flex-direction: column;
-    gap: $space-3;
+    // A touch more room between fields in the landing demo (minimal mode only).
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)